Include/cpython/optimizer.h

```diff
@@ -36,12 +36,29 @@ typedef struct {
     uint64_t operand;  // A cache entry
 } _PyUOpInstruction;
 
+typedef struct _exit_data {
+    uint32_t target;
```

Comment:
IIUC the `target` field in `_PyUOpInstruction` is now an index into the exits array. Maybe rename it? (E.g. to `exit_index`.)

Reply:
It is still the target for `DEOPT_IF` exits though. A union perhaps?
```C
union {
    uint32_t target;
    uint16_t exit_index;
}
```